### PR TITLE
Add new auth service to pipelines

### PR DESCRIPTION
**Pull request by tej99**
# Motivation and Context

django-oauth-test is being replaced with ras-rm-auth-service. This Pr adds the new service into the pipelines.

# What has changed

Added ras-rm-auth-service to pipelines

# How to test?

fly pipelines and check the service is up and green

# Links

https://trello.com/c/1NVPXyuQ/388-re-implement-missing-auth-functionality-in-ras-rm-auth-service

### CODE REVIEW

**Review by MIslam92 — CHANGES_REQUESTED** [review]
You might have to add/do some changes to the `create_default_user.yml` , `get_database_uris.sh` and `health_check.sh` as well for the new `ras-rm-auth-service`

**Comment by tej99** [comment]
@MIslam92 Nice spot, create_default_user.yml and health_check.sh is not needed for rasrm-auth-service but updated the get_database_uris.sh 👍

**Review by MIslam92 — APPROVED** [review]
LGTM 👍

File: pipelines/deployment.yml [hunk 1]
```diff
@@ -1158,8 +1183,6 @@ jobs:
         EQ_URL: 'https://eq-test/session?token='
         JSON_SECRET_KEYS: ((ci_json_secret_keys))
         JWT_SECRET: ((ci_jwt_secret))
-        OAUTH_CLIENT_ID: 'ons@ons.gov'
-        OAUTH_CLIENT_SECRET: 'password'
         SECRET_KEY: ((ci_enrolement_code_encryption_key))
 
 - name: django-oauth2-test-ci-deploy
```
Comment (liamtoozer) [inline pipelines/deployment.yml:1199]: Do the existing django-oauth2 jobs need to be removed from here now? Or are they staying as they are until the service is removed completely?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍 Just a quick question about the existing oauth2 service.

File: pipelines/deployment.yml (hunk 1 above)
Comment (tej99) [reply pipelines/deployment.yml:1199]: Keeping them in until it's removed completely. Will have a clean up once we fully deprecate the django-oauth-test   repo.

**Pull request merged**